### PR TITLE
Add migration sdk

### PR DESCRIPTION
**Pull request by mahsamoosavi**
- Fixed some codes in greeter and redeem retryable contracts
- arbitrum/nitro-contracts added
- Things are now fully compatible with migration sdk

### CODE REVIEW

File: packages/demo-dapp-election/.env-sample [hunk 1]
```diff
@@ -1,5 +1,8 @@
-DEVNET_PRIVKEY="0x your key here"
+# This is a sample .env file for use in local development.
+# Duplicate this file as .env here
 
-# this is rinkarby, can use any Arbitrum chain
-L2RPC="https://rinkeby.arbitrum.io/rpc"
+# Your Private key
+DEVNET_PRIVKEY ='0x your key here'
```
Comment (gzeoneth) [inline packages/demo-dapp-election/.env-sample:5]: I think these extra spaces are going to break stuffs, better do a search and replace to remove them.

File: packages/eth-deposit/package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "hardhat": "^2.2.0"
   },
   "dependencies": {
-    "@arbitrum/sdk": "^2.0.9",
+    "@arbitrum/sdk": "^2.0.10",
```
Comment (gzeoneth) [inline packages/eth-deposit/package.json:15]: The latest is 2.0.13
https://www.npmjs.com/package/@arbitrum/sdk

**Review by gzeoneth — CHANGES_REQUESTED** [review]

File: packages/demo-dapp-election/.env-sample (hunk 1 above)
Comment (mahsamoosavi) [reply packages/demo-dapp-election/.env-sample:5]: done :)

**Review by gzeoneth — APPROVED** [review]
LGTM

**Pull request merged**